features/netsocket/TCPSocket.h

```diff
@@ -111,7 +111,9 @@ class TCPSocket : public Socket {
      *  @param data     Destination buffer for data received from the host
      *  @param size     Size of the buffer in bytes
      *  @return         Number of received bytes on success, negative error
-     *                  code on failure
+     *                  code on failure. If no messages are available to be received
```

Comment:
Conversely to the UDP case, this should be "no [more?] data is available", rather than introducing "messages".